skins/oasis/js/Tracking.js

```diff
@@ -424,23 +426,70 @@ jQuery(function ($) {
 				});
 			});
 		}
-		if ($topModule.length) {
-			$topModule.on('mousedown', '.top-wiki-article-thumbnail a', function (e) {
+
+		if ($exactWikiMatchModule.length) {
+			track({
+				action: Wikia.Tracker.ACTIONS.IMPRESSION,
+				category: category,
+				label: 'exact-wiki-match'
+			});
+
+			$exactWikiMatchModule.on('mousedown', 'a', function (e) {
 				var el = $(e.currentTarget);
 				track({
 					browserEvent: e,
 					category: category,
-					label: 'top-module-thumb-' + el.data('pos')
+					label: 'exact-wiki-match-' + el.data('event')
 				});
-			}).on('mousedown', '.top-wiki-article-text a', function (e) {
+			});
+		}
+
+		if ($fandomStoriesModule.length) {
+			track({
+				action: Wikia.Tracker.ACTIONS.IMPRESSION,
+				category: category,
+				label: 'fandom-stories'
+			});
+
+			$fandomStoriesModule.on('mousedown', '.fandom-story-link', function (e) {
 				var el = $(e.currentTarget);
 				track({
 					browserEvent: e,
 					category: category,
-					label: 'top-module-title-' + el.data('pos')
+					label: 'fandom-story-link-' + el.data('pos')
+				});
+			}).on('mousedown', '.fandom-story-image', function (e) {
+				track({
+					browserEvent: e,
+					category: category,
+					label: 'fandom-story-image-' + el.data('pos')
 				});
 			});
 		}
+
+		if ($topModule.length) {
+			track({
+				action: Wikia.Tracker.ACTIONS.IMPRESSION,
+				category: category,
+				label: 'top-wiki-articles'
+			});
+
+			$topModule.on('mousedown', '.top-wiki-article-link', function (e) {
+				var el = $(e.currentTarget);
```

Comment:
As a jQuery object, the variable should be prefixed with `$` per conventions. But do we actually need the variable, or can the label just be `label: 'top-wiki-article-link-' + $(e.currentTarget).data('pos')`?